pkgs/os-specific/linux/apparmor/default.nix

```diff
@@ -2,13 +2,16 @@
 , pkgconfig, which
 , flex, bison
 , linuxHeaders ? stdenv.cc.libc.linuxHeaders
-, pythonPackages
+, python3Packages
+, gawk
 , perl
 , swig
+, ncurses
 , pam
 }:
 
 let
+  python = python3Packages.python;
```

Comment:
and drop this

Reply:
Done, `pkgs.python3` and `pkgs.python3Packages.python` are redundant, didn't know about that. This is confusing a little bit :)